frontend/src/pages/overview/overview/ChainsOverview.js

```diff
@@ -320,6 +320,26 @@ export const ChainsOverview = ({ chainNames }) => {
             </CardContent>
           </Card>
         </Grid>
+        <Grid item xs={6} md={3}>
+          <Card elevation={0}>
+            <CardContent>
+              <Typography
+                align="left"
+                variant="body2"
+                color="text.secondary"
+                fontWeight={600}
+              >
+                Total Wallet Balance
+              </Typography>
+              <Typography align="left" variant="h6" color="text.primary">
+                $
+                {formatNumber(
+                  totalAvailableAmount + totalStakedAmount + totalPendingAmount
```

Comment:
We should also add IBC assets value to total balance

Reply:
Okay Aleem